src/main/scala/is/hail/expr/AST.scala

```diff
@@ -596,6 +596,26 @@ case class Apply(posn: Position, fn: String, args: Array[AST]) extends AST(posn,
 
         `type` = tNew
 
+      case "uniroot" =>
+        `type` = TFloat64
```

Comment:
delete. Type defined by looking up `uniroot` in function registry. I assume that is serving the purpose of getting the return type as well as checking the two input arguments for min and max are numeric.